Makefile

```diff
@@ -56,11 +56,29 @@ learn-ocaml.install:
 	    echo '  "$(f)" {"www/${f:static/%=%}"}' >>$@;)
 	@echo ']' >>$@
 
+# Generates up-to-date translation template for lang % from the sources
+LANGS = $(patsubst translations/%.po,%,$(wildcard translations/*.po))
+translations/$(LANGS:=.pot):
+	@for f in $(LANGS); do echo >> translations/$$f.po; done
+	@rm -f translations/*.pot
+	@DUMP_POT=1 ocp-build -j 1
+	@for f in $(LANGS); do \
+	  mv translations/$$f.pot translations/$$f.pot.bak; \
+	  msguniq translations/$$f.pot.bak > translations/$$f.pot; \
```

Comment:
Is GNU/gettext standard on MacOS X?

Reply:
I don't know really, but it certainly can be installed without too much effort (e.g. using [homebrew](https://formulae.brew.sh/formula/gettext)). It is, in any case and mot importantly, not needed in the normal build process, but only to synchronise the translations after a change of the strings in the source, or to generate a new translation template to start a new language.